.travis.yml

```diff
@@ -18,3 +18,8 @@ deploy:
   on:
     repo: SUSE/saltstack-netapi-client-java
     tags: true
+after_success:
+- scripts/publish_javadoc.sh
+env:
+  global:
+  - secure: aFeCOD180jlpw9yCqyKDXDuMrcVaNj607sCsIcFZ+rIYo1bMFWl5iLrQasQgtzE9jtNto6VYRq/BHWDR7qBSyk0qg2yWh+qhs7O/53m1XDD5DJryVg89IW4ynauCIeVNSfaLRzWRklphHeaLifNxBLd0v9hAG6z+skx65s7VzLM=
```

Comment:
Do you think we could use the same token here as we use above in the `deploy` section? Or is there any advantage in using two different access tokens that I am not aware of?